toml_serialization.nimble

```diff
@@ -7,6 +7,7 @@ description   = "Flexible TOML serialization [not] relying on run-time type info
 license       = "Apache License 2.0"
 skipDirs      = @["tests", "assets"]
 
+requires "https://github.com/nitely/nim-faststreams#768bbbdc7db4125afbd5d7da07e067903f29942b"
```

Comment:
this needs to be `require faststreams >= 0.5.0` once faststreams 0.5.0 is bumped+tagged